weblate/formats/ttkit.py

```diff
@@ -645,8 +645,8 @@ def is_translated(self):
     def is_fuzzy(self, fallback=False):
         """Check whether unit needs edit.
 
-        The isfuzzy on XLIFF is really messing up approved flag with fuzzy
-        and leading to various problems.
+        The isfuzzy on XLIFF is really messing up \"approved"\\ flag with fuzzy
```

Comment:
There is no need to escape quotes here:

```suggestion
        The isfuzzy on XLIFF is really messing up "approved" flag with fuzzy
```